pkg/validations/createcluster/createcluster.go

```diff
@@ -0,0 +1,86 @@
+package createcluster
+
+import (
+	"context"
+	"fmt"
+	"runtime"
+
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/gitops/flux"
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/pkg/providers"
+	"github.com/aws/eks-anywhere/pkg/validations"
+	"github.com/aws/eks-anywhere/pkg/workflows/interfaces"
+)
+
+type validationManager struct {
+	clusterSpec    *cluster.Spec
+	provider       providers.Provider
+	gitOpsFlux     *flux.Flux
+	cmdValidations interfaces.Validator
```

Comment:
instead of `cmdValidations`, `createValidations`?

Reply:
Updated this to `createValidations` and got rid of the interfaces.Validator reference